src/media-segment-request.js

```diff
@@ -290,7 +290,8 @@ const transmuxAndNotify = ({
     if (probeResult) {
       trackInfoFn(segment, {
         hasAudio: probeResult.hasAudio,
-        hasVideo: probeResult.hasVideo
+        hasVideo: probeResult.hasVideo,
+        isMuxed
```

Comment:
When messing around with renditions in safari I realized that we don't pass isMuxed up for ts segments. This is a problem because ts segments can contain video/audio and be appended to a "videoBuffer"  that has both codecs.